test/Microsoft.DotNet.TestFramework/Microsoft.DotNet.TestFramework.csproj

```diff
@@ -1,13 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <Description>Microsoft.DotNet.TestFramework Class Library</Description>
-    <VersionPrefix>$(CliVersionPrefix)</VersionPrefix>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <AssemblyOriginatorKeyFile>../../tools/Key.snk</AssemblyOriginatorKeyFile>
-    <SignAssembly>true</SignAssembly>
-    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
-    <NetStandardImplicitPackageVersion>2.0.0</NetStandardImplicitPackageVersion>
+
```

Comment:
This is the MicrosoftAspNetKey from arcade. You could switch to just using that.